src/modules/wrap-with-useeffect.ts

```diff
@@ -0,0 +1,31 @@
+import { selectedText, importMissingDependencies, activeFileName, allText } from "../editor";
+import { replaceSelectionWith } from "../code-actions";
+import { buildEffectHook } from "../snippet-builder";
+import { codeToAst, astToCode } from "../parsing";
+import { persistFileSystemChanges } from "../file-system";
+import * as t from '@babel/types';
+import { findPathInContext } from "../ast-helpers";
+
+
+export async function isInsideOfFunctionBody(text) {
+    try {
+        const allAST = codeToAst(allText());
+        const containerPath = findPathInContext(allAST, text);
+        return !!containerPath.findParent(parent => {
+            return t.isFunctionExpression(parent)
+        });
+    } catch (e) {
+        return false;
+    }
+}
+export async function wrapWithUseEffect() {
```

Comment:
![Codacy](https://app.codacy.com/assets/images/favicon.png) Issue found: [expected call-signature: 'wrapWithUseEffect' to have a typedef](https://app.codacy.com/gh/wix/vscode-glean/pullRequest?prid=5535229)